keps/sig-node/4540-strict-cpu-reservation/README.md

```diff
@@ -411,7 +405,7 @@ Longer term, we may want to require automated upgrade/rollback tests, but we
 are missing a bunch of machinery and tooling and can't do that now.
 -->
 
-We manually test it in our internal environment and it works.
+We use the feature in our internal environment and it works.
 
 ###### Is the rollout accompanied by any deprecations and/or removals of features, APIs, fields of API types, flags, etc.?
```

Comment:
Hmmm... the feature gate rename - that's something I'm not 100% confident will work nicely.

Reply:
Not sure what do you mean. CPUManager uses AlphaOptions gate for alpha stage policy options and BetaOptions gate for beta stage options, not new to this KEP.